include/libdnf/base/base.hpp

```diff
@@ -0,0 +1,41 @@
+/*
+Copyright (C) 2020 Red Hat, Inc.
+
+This file is part of libdnf: https://github.com/rpm-software-management/libdnf/
+
+Libdnf is free software: you can redistribute it and/or modify
+it under the terms of the GNU Lesser General Public License as published by
+the Free Software Foundation, either version 2 of the License, or
+(at your option) any later version.
+
+Libdnf is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU Lesser General Public License for more details.
+
+You should have received a copy of the GNU Lesser General Public License
+along with libdnf.  If not, see <https://www.gnu.org/licenses/>.
+*/
+
+#ifndef LIBDNF_BASE_BASE_HPP
+#define LIBDNF_BASE_BASE_HPP
+
+
+namespace libdnf {
+
+
+/// Instances of :class:`lindnf::Base` are the central point of functionality supplied by libdnf.
+/// An application will typically create a single instance of this class which it will keep for the run-time needed to accomplish its packaging tasks.
+/// :class:`.Base` instances are stateful objects owning a various data.
+///
+/// @replaces dnf:dnf/base.py:class:Base
```

Comment:
I would remove the @replaces string for now and add it back later when we remove the actual code from dnf.

Reply:
OK